src/actions/users.js

```diff
@@ -610,6 +610,37 @@ export function getMe() {
     };
 }
 
+export function updateServiceTermsStatus(serviceTermsId, accepted) {
+    return async (dispatch, getState) => {
+        const {data, error} = await dispatch(bindClientFunc(
+            Client4.updateServiceTermsStatus,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_REQUEST,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_SUCCESS,
+            UserTypes.UPDATE_SERVICE_TERMS_STATUS_FAILURE,
+            serviceTermsId,
+            accepted
+        ));
+        if (data && data.status === 'OK') {
+            const currentUser = getCurrentUser(getState());
+            dispatch({
+                type: UserTypes.RECEIVED_ME,
+                data: Object.assign({}, currentUser, {accepted_service_terms_id: JSON.parse(accepted) ? serviceTermsId : null}),
```

Comment:
`accepted` is just true or false, right? I don't think it needs a `JSON.parse()` around it